model/Licensing/Properties/licenseId.md

```diff
@@ -4,11 +4,26 @@ SPDX-License-Identifier: Community-Spec-1.0
 
 ## Summary
 
-TODO
+A licenseId provides a short, unique identifier to refer to a License or
+LicenseException.
 
 ## Description
 
-TODO
+A licenseId contains a human-readable, short-form license identifier for a
+License or LicenseException. It may include only letters, numbers, period
+(".") and hyphen ("-") characters.
+
+For a ListedLicense or ListedLicenseException, the licenseId will be as
+specified on the [SPDX License List](https://spdx.org/licenses) for the
+particular license or exception.
+
+For a CustomLicense, the licenseId must begin with the prefix `LicenseRef-`
+and must be unique within the applicable SPDX namespace.
```

Comment:
licenseId -> short form license ID
the short form license ID may be preceded by an SPDX namespace or a fully qualified URI prefix.

Reply:
Good point @goneall -- thanks, I'll update this.